app/models/game.rb

```diff
@@ -1,8 +1,11 @@
 class Game < ApplicationRecord
+	belongs_to :user
+
+	validates :name, presence: true
+
 #I'm not going to try to initialize this just yet in the rails side of things. First I want to get a good idea of how I can style an array on a page!
```

Comment:
I think this might be where js ends up coming in to inject the piece images into the table I wrote to display the game from it's position as stored in the game model. Good start though!